src/main/java/by/exadel/internship/controller/SchedulingController.java

```diff
@@ -0,0 +1,56 @@
+package by.exadel.internship.controller;
+
+
+import by.exadel.internship.annotation.AdminAccessControl;
+import by.exadel.internship.annotation.SuperAdminAccessControl;
+import by.exadel.internship.dto.timeForCall.UserTimeSlotWithUserIdDTO;
+import by.exadel.internship.dto.UserDTO;
+import by.exadel.internship.service.SchedulingService;
+import io.swagger.annotations.Api;
+import io.swagger.annotations.ApiOperation;
+import lombok.RequiredArgsConstructor;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+import java.util.UUID;
+
+@RestController
+@RequiredArgsConstructor
+@RequestMapping("/scheduling")
+@Api(tags = "Endpoints for Scheduling")
+public class SchedulingController {
+
+    private final SchedulingService schedulingService;
+
+    @SuperAdminAccessControl
+    @PostMapping()
+    @ApiOperation("Save Users free time")
+    public void saveUserTime(@RequestBody UserDTO userDTO) {
```

Comment:
Proposal: move this endpoint to UserController under url POST `/users/{userId}/time-slots`
@SergeyKoval @viktarkolbik what do you think?

Reply:
I think it's logical. but i don't know shouuld i move saveUserTime method to UserService?